tests/app/clients/test_freshdesk.py

```diff
@@ -220,3 +221,13 @@ def match_json(request):
                 )
             ).send_ticket()
             assert response == 201
+
+
+def test_send_ticket_freshdesk_integration_disabled(mocker, notify_api: Flask):
+    with set_config_values(notify_api, {"FRESH_DESK_ENABLED": "False"}):
```

Comment:
Removed the env var set in `conftest.py` since we've got a default value already in `config.py` and this test is a bit of a one off.